src/App.tsx

```diff
@@ -86,7 +96,9 @@ function App() {
                   />
                 </h1> */}
                 {/* TODO: move to styling file */}
-                <DomainsGlobal domain={'ROOT'} />
+                <div style={{ position: 'absolute', top: '0' }}>
```

Comment:
If we're using most of our styling in SCSS classes, can we be consistent about that and not use inline styles?

Reply:
ok

Reply:
Will be addressed in another pull request